server_admin/topics/clients/oidc/con-audience.adoc

```diff
@@ -0,0 +1,123 @@
++
+.The untrusted service serves the request to the client application but also keeps the token.
+
+. The untrusted service invokes a trusted service with the token. Invocation is not successful because the trusted service checks the audience on the token and find that its audience is only for the untrusted service. This is expected behavior and security is not broken.
+
+If the client wants to invoke the trusted service later, it must obtain another token by reissuing the SSO login with `scope=<trusted service>`. The returned token will then contain the trusted service as an audience:
+
+[source,json]
+----
+"audience": [ "<trusted service>" ]
+----
+Use this to invoke the `<trusted service>`.
+
+===== Setup
+
+When setting up audience checking:
+
+* Ensure that services are configured to check audience on the access token sent to them by adding the flag `_verify-token-audience_` in the adapter configuration. See link:{adapterguide_link}#_java_adapter_config[Adapter configuration] for details.
+
+* Ensure that access tokens issued by {project_name} contain all necessary audiences. Audiences can be added using the client roles as described in the <<_audience_resolve, next section>> or hardcoded as described in <<_audience_hardcoded, below>>.
+
+[[_audience_resolve]]
+===== Automatically add audience
+
+An _Audience Resolve_ protocol mapper is defined in the default client scope _roles_. The mapper checks for clients that have at least one client role available for the current token. The client ID of each client is then added as an audience. This is useful if your service (usually bearer-only) clients rely on client roles.
+
+For example, for a bearer-only client and a confidential client,
+you can use the access token issued for the confidential client to invoke the bearer-only client REST service. The bearer-only client will be automatically added as an audience to the access token issued for the confidential client if the following are true:
+
+* The bearer-only client has any client roles defined on itself.
+
+* Target user has at least one of those client roles assigned.
+
+* Confidential client has the role scope mappings for the assigned role.
+
+[NOTE]
+====
+If you want to ensure that the audience is not added automatically, do not configure role scope mappings directly on the confidential client. Instead, you can create a dedicated client scope that contains the role scope mappings for the client roles of your dedicated client scope. 
+
+Assuming that the client scope is added as an optional client scope to the confidential client, the client roles and the audience will be added to the token if explicitly requested by the `scope=<trusted service>`  parameter.
+====
+
+[NOTE]
+====
+The frontend client itself is not automatically added to the access token audience. This allows for easy differentiation between
```

Comment:
suggestion: remove "itself"
Change "This allows for easy differentiation"  to "Since the client is omitted, it is easy differentiate.."

Reply:
@briandooley because you deleted the old file audience.adoc and created a new one, I can't tell what is change.  
So please checkout the original file, move the new content in there and rename the file.